frontend/components/doctor/appointment-reminders.tsx

```diff
@@ -0,0 +1,73 @@
+import { Box, Center, Flex, Text } from "@chakra-ui/react";
+import { Appointment, APPOINTMENT_STATUS, SECTION } from "@frontend/models/appointment";
+import moment from "moment";
+
+export default function AppointmentReminders({
+    appointmentList,
+    section,
+}: {
+    appointmentList: Appointment[];
+    section: string;
+}) {
+    function borderColor(status: string) {
+        switch (status) {
+            case APPOINTMENT_STATUS.CONFIRMED:
+                return "green.200";
+            case APPOINTMENT_STATUS.PENDING:
+                return "orange.200";
+            case APPOINTMENT_STATUS.DECLINED:
+                return "red.200";
+        }
+    }
+
+    function dateDisplay(appointment: Appointment) {
+        switch (appointment.date) {
+            case moment().format("YYYY-MM-DD"):
+                return "Today";
+            case moment().add(1, "days").format("YYYY-MM-DD"):
+                return "Tomorrow";
+            case moment().add(-1, "days").format("YYYY-MM-DD"):
+                return "Yesterday";
+            default:
+                return moment(appointment.date).format("ddd. MMMM DD");
+        }
+    }
+    return (
+        <Box mr={4}>
+            {appointmentList.length ? (
+                appointmentList.map(appointment => (
+                    <Box
+                        key={appointment.appointmentId}
+                        borderWidth={2}
+                        borderColor={section === SECTION.PAST ? "gray.200" : borderColor(appointment.status)}
+                        my={2}
+                        borderRadius={5}
+                        py={2}
+                        px={4}
+                        fontSize="lg">
+                        <Flex>
```

Comment:
Probably should remove the Flex on line 48. Looks a bit better in mobile view. Maybe mess with the fontSize

![image](https://user-images.githubusercontent.com/57551054/161142066-07d29e13-23f1-4adf-b8bc-e62405ff93c4.png)

Reply:
I fixed it so that it goes on 2 lines on smaller screens but stays 1 line for bigger ones so that it doesn't look too empty